src/Microsoft.Health.Dicom.Blob/DicomBlobContainerOptions.cs

```diff
@@ -5,15 +5,26 @@
 
 using System.ComponentModel.DataAnnotations;
 
-namespace Microsoft.Health.Dicom.Functions.Configuration;
+namespace Microsoft.Health.Dicom.Blob;
 
-internal class DicomBlobContainerConfiguration
+/// <summary>
+/// Represents the various Azure Blob Containers used by the DICOM server.
+/// </summary>
+public sealed class DicomBlobContainerOptions
```

Comment:
I have been trying this configuration format in functions for listing our various containers, and if we like it enough I can migrate the server too